dentapp/src/components/TextFeedback.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import Container from "react-bootstrap/Container";
+import SuccessIcon from "../assets/Success.svg";
+
+const TextContainer = styled.div`
+  color: ${(props) => props.color};
+  ${(props) => props.error && "text-decoration: underline;"}
+`;
+
+class TextFeedback extends Component {
+  state = {
+    status: this.props.status,
+    text: this.props.text,
+    title: this.props.title,
+  };
```

Comment:
Is state necessary? I don't see is beign used in the component. If that is the case, change it to Functional Component